lib/movie_details.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'styles.dart';
+
+class MovieDetails extends StatelessWidget {
+  MovieDetails({
+    super.key,
+    required this.movieOverview,
+  });
+
+  final String movieOverview;
+
+  static const String descriptionText = 'Description';
+
+  TextStyle descriptionTextStyle = const TextStyle(
+    fontSize: fontText,
+    color: Colors.white54,
+  );
+
+  static const int maxLinesOverviewText = 10;
+
+  static const double paddingTopBetweenDescriptionOverview = 12;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: double.infinity,
+      child: ListView(
```

Comment:
This could be a `Column`, as you're instantiating this widget inside a `SingleChildScrollView`.

Reply:
Done!